snap/snapcraft.yaml

```diff
@@ -43,10 +43,8 @@ apps:
       - etc-grafana-agent
       - proc-sys-kernel-random
 architectures:
-  - build-on: [amd64]
-  - build-on: [amd64, arm64]
-    build-for: [arm64]
-
+  - build-on: amd64
+  - build-on: arm64
```

Comment:
This diff is a revert of a commit that was accidentally pushed directly to main.